kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -171,6 +157,22 @@ const List: React.FC<TopicsListProps> = ({
     },
     [setTopicsSearch, history, pathname, perPage, page]
   );
+  const deleteOrPurgeConfirmationHandler = React.useCallback(() => {
+    if (confirmationModal === 'deleteTopics') {
+      deleteTopics(clusterName, Array.from(tableState.selectedIds));
+    } else {
+      clearTopicsMessages(clusterName, Array.from(tableState.selectedIds));
```

Comment:
I think it would be better to store `Array.from(tableState.selectedIds)`  in a variable above the conditions and use it inside the condition blocks. That way we'll prevent code duplication.

```
const selectedIds = Array.from(tableState.selectedIds);

if (confirmationModal === 'deleteTopics') {
  deleteTopics(clusterName, selectedIds);
} else {
  clearTopicsMessages(clusterName, selectedIds);
}
```

Looks cleaner to me.